src/validations/index.js

```diff
@@ -27,9 +27,17 @@ function runValidation(validation, param, pkgInfo, errs) {
         });
 }
 
+function skipValidation(validation) {
```

Comment:
@inikulin : I need your help on how to render the skipped text. Maybe current code is not the most appealing solution.

Reply:
I think that maybe it's worth aborting publishing completely in this case pushing user to either remove this validation from config or update npm?

Reply:
My initial intent was to make a non breakable change for existing consumers in the way that if vulnerability check was green before , it is still green after this upgrade. 

What you suggest is to always fail the vulnerability check if npm version is not ok.

In this case I have to revert the 'skippable' feature and just throw an error with adequate error message. I agree It will simplify the overall changes. 

Please let me know if I have to go this way.